index.html

```diff
@@ -0,0 +1,551 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="HTML and CSS" />
+    <meta name="keywords" content="Portfolio Mobile Version" />
+    <meta name="author" content="Itumeleng Rantsane" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="../images/favicon-32x32.png"
+    />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=UA-199543711-1"
+    ></script>
+
+    <title>Itumeleng Rantsane | Portfolio</title>
+  </head>
+
+  <body>
+    <div class="hero">
+      <header class="header bg-white">
+        <nav class="navbar flex">
+          <a href="#" class="navbar__link link fz-20 fw-700 lh-20">I.R</a>
+          <div class="navbar__menu">
+            <ul class="navbar__menu__ul ul flex">
+              <li class="navbar__menu__li">
+                <a
+                  href="#works"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Portfolio</a
+                >
+              </li>
+              <li class="navbar__menu__li">
+                <a
+                  href="#about"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >About</a
+                >
+              </li>
+              <li class="navbar__menu__li">
+                <a
+                  href="#contact"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Contact</a
+                >
+              </li>
+            </ul>
+          </div>
+          <img
+            class="navbar__hamb"
+            src="images/hamburger.png"
+            alt="hamburger icon"
+          />
+        </nav>
+      </header>
+
+      <section class="head flex">
+        <h1 class="head__title fz-40 fw-700 lh-52">
+          I'm Tumi <span>Glad to see you!</span>
+        </h1>
+        <p class="head__dsc fz-16 fw-400 lh-24">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <a href="#works" class="head__link link fz-16 fw-500 lh-24 clr-p-400"
+          >My Projects</a
```

Comment:
- [ ] The text-content here is supposed to be `LET'S CONNECT` and not `MY PROJECTS` Kindly consider changing that and place everything where they are supposed to be placed  :smile: 

![image](https://user-images.githubusercontent.com/97100091/201145051-95d760a7-f2a8-4d51-a7f5-f08802832dda.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201146241-351ab079-ae47-4ef1-9592-a2975137f87a.png)